tests/test_xet_utils.py

```diff
@@ -0,0 +1,222 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from huggingface_hub import constants
+from huggingface_hub.utils._xet import (
+    XetMetadata,
+    _fetch_xet_metadata_with_url,
+    parse_xet_headers,
+    refresh_xet_metadata,
+)
+
+
+def test_parse_valid_headers_minimal() -> None:
+    headers = {
+        constants.HUGGINGFACE_HEADER_X_XET_ENDPOINT: "https://xet.example.com",
+        constants.HUGGINGFACE_HEADER_X_XET_ACCESS_TOKEN: "xet_token_abc",
+        constants.HUGGINGFACE_HEADER_X_XET_EXPIRATION: "1234567890",
```

Comment:
Let's not use constants in this test module to make the expected headers more readable (and also detect issue if constant value is updated without a good reason)